spid-spring-integration/src/main/java/it/italia/developers/spid/integration/util/SPIDIntegrationUtil.java

```diff
@@ -172,7 +184,7 @@ public Credential getCredential() {
 		credential.setEntityCertificate(certificate);
 		credential.setPrivateKey(pk);
 
-		log.info("Private Key" + pk.toString());
+		// log.info("Private Key" + pk.toString());
```

Comment:
There shouldn't be commented-out lines left in production code. Besides, a private key being sensible data, maybe it could be logged at least at DEBUG level?

```suggestion
		log.debug("Private Key: {}", pk);
```

Reply:
I do not agree. I think this is still dangerous and to be avoided: private keys should never be logged. I removed the commented code altogether.

Reply:
Cool, probably it's the best.